tsi_app/m85/src/main.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+#include <stdio.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/logging/log_ctrl.h>
+#include <zephyr/logging/log_output.h>
+
+#define DATA_MAX_DLEN 8
+#define LOG_MODULE_NAME m85 
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+int main(void)
+{
+
+	/* TSI banner */
+	printf("!! ------------------------------------------ !! \n");
```

Comment:
Can we define this as strings ? Is this a standard practice for the for a boot banner.